lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -117,6 +117,14 @@ sub ensure_size {
     return $size;
 }
 
+sub hidden {
+    # 1 if asset should not be linked from the web UI (as set by
+    # 'hide_asset_types' config setting), otherwise 0
+    my ($self) = @_;
+    my @types = split(/ /, $app->config->{global}->{hide_asset_types});
+    grep { $_ eq $self->type } @hides ? return 1 : return 0;
```

Comment:
I think this line has a name in 'the list of stupid code constructs' :)

return grep { $_ eq $self->type } @hides;